include/rocksdb/env.h

```diff
@@ -152,6 +152,15 @@ class Env {
                                      unique_ptr<RandomAccessFile>* result,
                                      const EnvOptions& options)
                                      = 0;
+  // These values match Linux definition
```

Comment:
Could you also add a link here to either linux kernel code or documentation mentioning lifetimes like this?

Reply:
I'll add a link